upgrade/UPGRADE-v10.0.0-dev.md

```diff
@@ -167,6 +167,7 @@ All the changes are considered as backwards incompatible.
         +        DocBlockParser $docBlockParser
              ) {
         ```
+- upgrade `ramsey/uuid` to version `^4.3.1` ([#2451](https://github.com/shopsys/shopsys/pull/2451))
```

Comment:
this is not necessary as the library is installed as the framework dependency. Users don't need to do anything and get the right version installed after upgrade of the shopsys/framework package